packages/react/src/themes/teams/components/Tree/treeTitleStyles.ts

```diff
@@ -0,0 +1,18 @@
+import { ICSSInJSStyle } from '../../../types'
+import { pxToRem } from '../../../../lib'
+import getBorderFocusStyles from '../../getBorderFocusStyles'
+
+const treeTitleStyles = {
+  root: ({ variables, theme: { siteVariables } }): ICSSInJSStyle => ({
+    padding: `${pxToRem(1)} 0`,
```

Comment:
Can we move this to variables?

```suggestion
    padding: v.padding,
```